Assets/MRTK/Providers/LeapMotion/Editor/Microsoft.MixedReality.Toolkit.Providers.LeapMotion.Editor.asmdef

```diff
@@ -15,5 +15,7 @@
     "overrideReferences": false,
     "precompiledReferences": [],
     "autoReferenced": true,
-    "defineConstraints": []
+    "defineConstraints": [],
+    "versionDefines": [],
```

Comment:
I think some of these asmdefs were edited in Unity 2019 (this is a new field in 2019). Should we make sure these are serialized in 2018 before we check in?

Reply:
Yes please!